<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushPalette</class>
 <widget class="QWidget" name="BrushPalette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>368</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="GroupedToolButton" name="presetAdd">
       <property name="toolTip">
        <string>Add a new brush preset</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>theme:list-add.svg</normaloff>theme:list-add.svg</iconset>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GroupedToolButton" name="presetSave">
       <property name="toolTip">
        <string>Overwrite selected preset</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff>theme:document-save.svg</normaloff>theme:document-save.svg</iconset>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupCenter</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GroupedToolButton" name="presetDelete">
       <property name="toolTip">
        <string>Delete selected preset</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete">
         <normaloff>theme:edit-delete.svg</normaloff>theme:edit-delete.svg</iconset>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupRight</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="brushPaletteView">
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupedToolButton</class>
   <extends>QToolButton</extends>
   <header>widgets/groupedtoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
